app/models/foreman_openscap/scap_content.rb

```diff
@@ -1,27 +1,23 @@
 require 'digest/sha2'
-require 'openscap/ds/sds'
-require 'openscap/source'
-require 'openscap/xccdf/benchmark'
 
 module ForemanOpenscap
   class DataStreamValidator < ActiveModel::Validator
     def validate(scap_content)
       return unless scap_content.scap_file_changed?
 
-      allowed_type = 'SCAP Source Datastream'
-      if scap_content.source.try(:type) != allowed_type
-        scap_content.errors[:base] << _("Uploaded file is not #{allowed_type}.")
+      api = ProxyAPI::Openscap.new(:url => SmartProxy.with_features('Openscap').first.url) rescue nil
```

Comment:
the `rescue nil` is to prevent a situation where there is no smart proxy with with Openscap feature.
Not sure it's the perfect solution. Any other ways to prevent error? I just want to ensure we've got a smart proxy with Openscap feature and return an error if not.